### PR TITLE
[[ Bug 17369 ]] Maintain showIcon state of toolbar icons

### PR DESCRIPTION
**Pull request by livecodeali**

### CODE REVIEW

File: Toolset/palettes/menubar/revmenubar.livecodescript [hunk 1]
```diff
@@ -710,13 +710,21 @@ on updateButtonHilite pTargetName, pValue
...
    set the label of tTargetLongID to revMenubarButtonText(pTargetName, pValue)
    if pValue then
       set the icon of tTargetLongID to the cHighlightedIcon of tTargetLongID
    else
       set the icon of tTargetLongID to the cIcon of tTargetLongID
    end if
+   
+   if not tShowIcon then
+      set the showIcon of tTargetLongID to false
+   end if
```
Comment (peter-b) [inline Toolset/palettes/menubar/revmenubar.livecodescript]: Could you unilaterally `set the showIcon of tTargetLongID to tShowIcon` here, without the test?

Comment (livecodeali) [inline Toolset/palettes/menubar/revmenubar.livecodescript]: Yep!

**Comment by runrevmark** [comment]
@livecode-vulcan review ok be54250

**Comment by livecode-vulcan** [comment]
💙 review by @runrevmark ok be54250acd34f26df1f3b111e1cf930277304c17

**Pull request merged**